ISSUE_TEMPLATE/RFC_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 💭 Request for comments
+about: Describe an idea and request comments in this project
+title: '[RFC]'
+labels: enhancement
+assignees: ''
+---
+**What are your requirements? What kind of business use-case are you trying to solve?**
+A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
+
+**What is the problem? What is preventing you from meeting the requirements?**
```

Comment:
Should this come before **What are your requirements? What kind of business use-case are you trying to solve?**?

Reply:
Right on. Swapped.